Makefile

```diff
@@ -235,6 +235,7 @@ proto: ## Build proto (Protocol Buffers) files
 	$(PROTOC) --proto_path=pkg/kv --go_out=pkg/kv --go_opt=paths=source_relative secondary_index.proto
 	$(PROTOC) --proto_path=pkg/gateway/multiparts --go_out=pkg/gateway/multiparts --go_opt=paths=source_relative multipart.proto
 	$(PROTOC) --proto_path=pkg/actions --go_out=pkg/actions --go_opt=paths=source_relative actions.proto
+	$(PROTOC) --proto_path=pkg/auth/model --go_out=pkg/auth/model --go_opt=paths=source_relative model.proto
```

Comment:
It looks like it contains changes from the previous PR too. Commit message ("draft for the models") also suggests it. Are you on the latest `master`?

Reply:
On the auth models pr we didn't add it, correct me if I'm wrong. I didn't see makefile in that PR.